doc/Language/modules.pod6

```diff
@@ -1069,8 +1069,20 @@ fez upload
 >>= Hey! You did it! Your dist will be indexed shortly
 =end code
 
+=head3 Versioning and fez
+
+=item To prevent changes to a versioned distribution (remember a distribution may contain more than
+one module), only B<one> upload of a distribution per version is permitted. If a developer makes changes to 
+code in a repository, the changes will not be reflected in the C<fez> system. So changes to code must be
+accompanied by a bump in the version number and the new version uploaded to C<fez>.
+
 =head2 Upload your module to p6c
 
+The Raku community is migrating to the I<fez> ecosystem (see above). For developers
+who have modules on the I<p6c> ecosystem and migrated them to I<fez>, please remove them
+from the I<p6c> ecosystem. If you do not have the commit bit, or ping someone on IRC (#raku at libera.chat) to
```

Comment:
Looks like a part of the sentence is missing here? Something like "the commit bit, open a pull request for the ecosystem repo, or ping...".

Reply:
Good catch

Reply:
if you can address this, we can merge this change

Reply:
@finanalyst - can you fix this item?